.github/workflows/ci-schema-validation.yml

```
@@ -0,0 +1,61 @@
+        uses: actions/checkout@v4
+
+      - name: "Gradle Wrapper 무결성 검증"
+        uses: gradle/actions/wrapper-validation@v4
+
+      - name: "JDK ${{ matrix.java-version }} 설정"
+        uses: actions/setup-java@v4
+        with:
+          distribution: ${{ matrix.distribution }}
+          java-version: ${{ matrix.java-version }}
+          cache: gradle
+          kotlin-version: ${{ matrix.kotlin-version }}
```

actionlint (1.7.10)

[error] input "kotlin-version" is not defined in action "actions/setup-java@v4". available inputs are "architecture", "cache", "cache-dependency-path", "check-latest", "distribution", "gpg-passphrase", "gpg-private-key", "java-package", "java-version", "java-version-file", "jdkFile", "job-status", "mvn-toolchain-id", "mvn-toolchain-vendor", "overwrite-settings", "server-id", "server-password", "server-username", "settings-path", "token"

(action)